Perform test

abiogenetic opened the pull request.

brzzw5k committed:
feat: Doolittle Factorization + minor ci changes (#2)

* update readme

* feat: Doolittle Factorization + minor ci tests

* update gitignore vscode

* fix

* ci install requirements

* requirements pre-commit

abiogenetic committed:
performance tests

brzzw5k commented on tests/performance_tests_doolittle.py:
```diff
@@ -0,0 +1,15 @@
+import numpy as np
```
tests filename should start with test_
I'd suggest renaming to test_doolittle_factorization_performance

brzzw5k commented on tests/performance_tests_doolittle.py:
```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import pytest
+from src.doolittle_factorization import DoolittleFactorization
+import time
```
unused import

abiogenetic committed:
performance tests

abiogenetic committed:
Add files via upload

abiogenetic committed:
Changes to be committed:
	new file:   tests/test_performance_doolittle.py

Changes not staged for commit:
	modified:   src/doolittle_factorization.py
	modified:   tests/test_performance_doolittle.py

abiogenetic committed:
On branch perform_test
 Your branch and 'origin/perform_test' have diverged,
 and have 1 and 1 different commits each, respectively.

Changes to be committed:
	modified:   src/doolittle_factorization.py
	modified:   tests/test_doolittle_factorization.py
	modified:   tests/test_performance_doolittle.py

abiogenetic committed:
Merge branch 'perform_test' of https://github.com/brzzw5k/prir-projekt-1 into perform_test

On branch perform_test
Your branch and 'origin/perform_test' have diverged,
and have 2 and 1 different commits each, respectively.
All conflicts fixed but you are still merging.
Changes to be committed:
	modified:   tests/test_performance_doolittle.py

abiogenetic committed:
Merge branch 'main' into perform_test
